eventually/vote/views.py

```diff
@@ -0,0 +1,337 @@
+"""
+    Views module
+    ============
+"""
```

Comment:
add the blank line after the end of docstring.

Reply:
+